src/web/theme/modules/Header/Navigation/Menu/Menu.scss

```diff
@@ -0,0 +1,94 @@
+.menu {
+  display: flex;
+  flex-direction: row;
+  position: relative;
+  top: 0;
+  transition: transform 0.2s ease-in-out;
+  z-index: 1;
+
+  &.close {
+    transform: translateX(calc(-100% + 3em));
+
+    button.menu__button.button {
```

Comment:
This selector is too specific. Could you think of another way to achieve styling using simpler rules?